ui/.eslintrc

```diff
@@ -1,13 +1,14 @@
 {
   "root": true,
   "parser": "babel-eslint",
-  "plugins": ["react-hooks"],
-  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["react-hooks", "prettier"],
+  "extends": ["airbnb",  "prettier"],
```

Comment:
I believe you can just append [eslint-plugin-prettier/recommended](https://github.com/prettier/eslint-plugin-prettier?tab=readme-ov-file#configuration-legacy-eslintrc) to extends and remove `prettier` from everywhere else (plugins, extends and rules). That one should take care of all that

Reply:
I think you are correct! Let me check and come back !